src/App.js

```diff
@@ -4,19 +4,40 @@ import Nav from './Nav'
 import MovieContainer from './MovieContainer'
 import MovieDetails from './MovieDetails'
 import './App.css';
-import movieData from './movieData';
-import movieDetails from './movieDetailData';
 class App extends Component {
   constructor() {
     super();
     this.state = {
-      movieList: movieData.movies,
+      movieList: [],
       currentMovie: ""
     }
   }
 
+  componentDidMount = () => {
+    fetch("https://rancid-tomatillos.herokuapp.com/api/v2/movies")
+      .then(response => {
+        if (!response.ok) {
+          throw Error()
+        } else {
+          return response.json()
+        }
+      })
+      .then(data => this.setState({movieList: data.movies}))
+      .catch(err => console.log(err))
```

Comment:
Consider doing some error handling that the user can see, rather than just console logging